### PR TITLE
fix typo mistake from matchs to matches.

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request
fixes: https://github.com/cmss13-devs/cmss13/issues/3795

# Explain why it's good for the game

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
spellcheck: fix typo mistake from matchs to matches.
/:cl:

### CODE REVIEW

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
use length(), not .len

**Comment by Huffie56** [comment]
> use length(), not .len

image
not working.

**Comment by Zonespace27** [comment]
https://www.byond.com/docs/ref/#/proc/length

**Comment by Huffie56** [comment]
> https://www.byond.com/docs/ref/#/proc/length

image
this seem to work. atleast it don't crash...

File: code/game/objects/items/storage/fancy.dm [hunk 1]
```diff
@@ -299,6 +299,15 @@
 	var/light_chance = 70 //how likely you are to light the match on the book
 	var/burn_chance = 20 //how likely you are to burn yourself once you light it
 
+/obj/item/storage/fancy/cigar/matchbook/get_examine_text(mob/user)
```
Comment (Zonespace27) [inline code/game/objects/items/storage/fancy.dm:302]: why are you duplicating `/obj/item/storage/fancy/get_examine_text(mob/user)` ?

File: code/game/objects/items/storage/fancy.dm [hunk 2]
```diff
@@ -299,6 +299,15 @@
...
+	. = ..()
+	if(length(contents) <= 0)
```
Comment (Zonespace27) [inline code/game/objects/items/storage/fancy.dm:304]:
```suggestion
	if(!length(contents))
```
0 is falsey

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/storage/fancy.dm (hunk 1 above)
Comment (Huffie56) [reply code/game/objects/items/storage/fancy.dm:302]: because i was thinking the second will override the first when it is used....
but it's not.
help.

**Review by Zonespace27 — APPROVED** [review]
Lgtm

**Pull request merged**